meta-jsdelivr/recipes-jsdelivr/jsdelivr-scripts/files/jsdelivr-updateContainer.sh

```
@@ -1,80 +1,111 @@
+           while :; do  sleep 2; done
+       fi
 
    fi
 
 
 
    if [ -f /tmp/updateFlag/JSDELIVR.RESET ]; then
       echo "Erase container update" > /dev/tty4
-      dd if=/dev/zero of=/dev/mmcblk0p3 bs=10M count=1
+
+      docker stop $(docker ps -a -q)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)